api-reference/receipts/endpoints.markdown

```diff
@@ -5,18 +5,26 @@ layout: reference
 
 # Endpoints
 
-- [GET Service Index](#endpoint-service-index)
-- [GET Schemas](#endpoint-schemas)
-- [GET Receipt Status by Receipt ID](#endpoint-get-receipt-status)
-- [Receipts with Data (eReceipts)](#eReceipts)
+- [Definitions of Resources](#definitions-of-resources)
+- [General](#general)
+  - [GET Service Index](#endpoint-service-index)
+  - [GET Receipt Status by Receipt ID](#endpoint-get-receipt-status)
+- [eReceipts](#ereceipts)
+  - [GET Schemas](#endpoint-schemas)
   - [POST Receipts](#endpoint-post-a-receipt)
   - [GET Receipts by User ID](#endpoint-get-receipts-by-userid)
   - [GET Receipts by Receipt ID](#endpoint-get-a-receipt-by-id)
   - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image)
-- [Receipt Images (Image-Only Receipts)](#image\-only-receipts)
-  - [POST Receipt Images](#endpoint-post-a-receipt-image)
-  - [GET Receipt Images by User ID](#endpoint-get-receipt-images-by-userid)
-  - [GET Receipt Image by ID](#endpoint-get-a-receipt-image-by-id)
+- [Image-Only Receipts](#image-only-receipts)
+  - [POST Image-Only Receipts](#endpoint-post-an-image-only-receipt)
+  - [GET Image-Only Receipts by User ID](#endpoint-get-image-only-receipts-by-userid)
+  - [GET Image-Only Receipt by Receipt ID](#endpoint-get-an-image-only-receipt-by-id)
+  - [GET Receipt Image by Receipt ID](#endpoint-get-receipt-image-image-only)
+
+#### Definitions of Resources
+
+- *__eReceipt__* - A schema-enforced resource with data.
```

Comment:
Could we clarify that eReceipts also have an image? Perhaps:

> *__eReceipt__* - A schema-enforced data resource. Accompanied with an image that's either sent by the user or generated from the data resource.

Reply:
👍